yolapy/models/user.py

```diff
@@ -0,0 +1,54 @@
+from yoconfig import get_config
+from yolapy.services import Yola
+
+
+class User(object):
+
+    """Yola User."""
+
+    def __init__(self, active=False, deleted=None, email=None, id=None,
+                 name=None, partner_id=None, preferences=None,
+                 signup_date=None, surname=None):
```

Comment:
My only concern is the duplication between kwarg and setting the attribute. If we add an attribute we have to remember to add it in both places. Which is fine if there are only a couple, but this a long list. What about accepting `**kwargs` and setting them with `setattr`? You could validate they match a list of acceptable attribute names that you store on the class or something.